data/faq/student_obligations.md

```diff
@@ -4,6 +4,12 @@ We expect you to:
 * Blog at least once per week (this will be aggregated on
 [blog.coala.io](https://blog.coala.io)).
 * Participate in weekly meetings with your mentor.
-* Get stuff merged, before the deadlines!
-* Get in contact with your mentors or the admins if any even remotely
-potential problems arise.
+* Get stuff merged, before the deadlines! If this, for whatever reason, looks
+to be problematic, talk to your mentor or an admin!
```

Comment:
I'm against this addition, getting stuff merged is mandantory. Period. If there's a problem with the goals that can be adjusted but this is not negotiable.